syntax = "proto2";

package e2e_noa;

message Behavior{
   enum FunctionId {
      NONE = 0;      
      NOA = 1;      
      APA = 2;      
      ACC = 3;      
      LKA = 4;      
      SUMMON = 5;      
      LKA_PLUS = 6;      
      CITY_NOA = 7;      
      MAPLESS_NOA = 8;
    }
    optional FunctionId function_id = 1 [default = NONE];
    optional uint32 tunnel = 2; // tunnel 
    optional bool auto_navi_lc_enable_status = 3; // for LCC AI driver
    optional bool system_break_stop = 4;
    optional double dynamic_headway = 5;
    optional bool traffic_light_func_enable = 6;
    optional double cruising_speed_limit = 7;
}